hw2/04-api.js

```diff
@@ -1,7 +1,31 @@
 /** Exercise 04 - API **/
```

Comment:
Code is easy to read, well formatted, and modular. Produces expected output and correctly formats population based on en-us number formats. Looks fantastic to me, no changes needed.